### ISSUE TITLE
Create Segment - Wholistic Endpoint Review

### ISSUE THREAD

**Issue by janiksam** [voucherifyio/voucherify-openapi#140]
## What

ReportedBy: @janiksam 

### Create Segment

**POST** `/segments`

#### Wholistic Endpoint Review

- [ ] Overall description
- [ ] Path parameters
- [ ] Headers
- [ ] Request description
- [ ] Request body
- [ ] Request examples
- [ ] Response description
- [ ] Response body
- [ ] Response example(s)
- [ ] Error response description
- [ ] Error response body
- [ ] Error response example(s)
- [ ] All parameters'
    - [ ] examples
    - [ ] defaults
    - [ ] allowed values
    - [ ] properties
    - [ ] formats
- [ ] Hyperlinks

**Comment by MaciekVoucheirfy** [comment]
Create segment - should there be a drop-down list and possibility to select ‘empty’ slots? Shouldn’t it be fixed and set to ‘static’ in Static and ‘auto-update’ in Dynamic? Because now you can select 'empty' field which causes API call to fail. Also when you switch to Dynamic we already have 'static' selected inside it at the beginning. It doesn't cause error, however, as in the code window it says 'auto update'.

Also, there is some bigger issue that causes the call to fail all the time - you cannot add customers in both cases. In static there are no fields to be filled with information whereas in dynamic filter object is lacking some information as well.

**Comment by janiksam** [comment]
You are right. The endpoint description is still being developed and is under 🚧 . There are may ways to create a dynamic segment and to describe this requires deep understanding and testing of the endpoint.
